samples/python/text_generation/chat_sample.py

```diff
@@ -10,8 +10,7 @@ def streamer(subword):
     print(subword, end='', flush=True)
     # Return flag corresponds whether generation should be stopped.
     # False means continue generation.
```

Comment:
```suggestion
    # Return flag corresponds whether generation should be stopped.
```
Same for samples/cpp/text_generation/chat_sample.cpp.

There are also
1. samples/cpp/text_generation/multinomial_causal_lm.cpp
2. samples/cpp/text_generation/prompt_lookup_decoding_lm.cpp
3. samples/cpp/text_generation/speculative_decoding_lm.cpp

Reply:
updated